src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
```

Comment:
The 'hidden' property should only be relevant to the 'Herbivore' class. There is no need to initialize it in the 'Animal' class constructor.